docs/watchdog.md

```diff
@@ -0,0 +1,73 @@
+# Screenshots-Watchdog Integration
+
+## Watchdog
+
+[Watchdog](https://github.com/mozilla/watchdog-proxy/) is a proxy service that send images to Microsoft's PhotoDNA service.  It is used by the Screenshots server.
+
+The shot images are submitted to Watchdog on their initial uploads and on edit saves.
+
+## Set It Up
+
+In order to use Watchdog, Screenshots needs to obtain a user ID and a key from Watchdog.  The required configuration values are:
+
+Config  | Environment Variable | Default Value | Notes
+------- | -------------------- | ------------- | -----
+Enabled | `ENABLE_WATCHDOG` | true | Even though the default is `true`, [`.env.dev`](https://github.com/mozilla-services/screenshots/blob/master/.env.dev) set it to `false`.
+User ID | `WATCHDOG_ID` | | A user id from Watchdog.  It is used for authentication.
+User Key | `WATCHDOG_KEY` | | The key for the above user id.
+Hash Algorithm | `WATCHDOG_AUTH_HASH_ALGORITHM` | sha256 | Algorithm for HMACs in [Hawk](https://github.com/hueniverse/hawk) authentication
+Submission URL | `WATCHDOG_SUBMISSION_URL` | | A full URL to the API endpoint where Watchdog is accepting requests from its consumers.  It should end with `/accept`.
+
+**Important!** If Watchdog is enabled, but missing any required configuration values or receives an invalid URL for the submission URL, it will stop the Screenshots server from starting.
+
+Other configuration values:
+
+Config  | Environment Variable | Default Value | Notes
+------- | -------------------- | ------------- | -----
+Positive Email | WATCHDOG_POSITIVE_EMAIL | | An optional list of email addresses to receive notifications of positive matches.
```

Comment:
How is this list denoted?  comma separated, or actual JS list or something else?

Reply:
It's semicolon delimited.  I'll add that to the doc before merging.  (The doc string at https://github.com/mozilla-services/screenshots/blob/master/server/src/config.js#L394 does mention it.  I'll file an issues to see if we'd want to generate a config markdown doc from `config.js`.)